ISSUE_TEMPLATE.md

```diff
@@ -1,72 +1,35 @@
-## my environment running ArangoDB
-I'm using the latest ArangoDB of the respective release series:
-- [ ] 2.8
-- [ ] 3.0
-- [ ] 3.1
-- [ ] 3.2
-- [ ] 3.3
-- [ ] self-compiled devel branch
+<!-- Please remove everything that does not apply -->
```

Comment:
Let's not do this... it didn't work in past. Maybe 1 out of 50 users removed superfluous things or ticket the boxes correctly in the old template. I don't see why this could improve now. Rather use HTML comments for all the examples we give to make them not render when submitted, see end of https://github.com/arangodb/arangodb/pull/4650#issuecomment-367630280